bindata/etcd/restore-pod.yaml

```diff
@@ -61,21 +58,18 @@ spec:
           fi
         fi
         
-        BUMP_ARGS=""
-        if [[ "${ETCD_ETCDCTL_RESTORE_ENABLE_BUMP}" == "true" ]]; then
-          echo "enabling restore bump"
-          BUMP_ARGS="--bump-revision 1000000000 --mark-compacted"
-        fi
-                
+        BUMP_REV=$(jq -r ".maxRaftIndex // 1000000000" -f "/var/lib/etcd-backup/revision.json")        
```

Comment:
need to still figure out how we can teach jq to do math, we need to add some percentage on top of the current maxRaftIndex to account for timing issues